templates/default/content/unfurl.tpl.php

```diff
@@ -8,8 +8,8 @@
     <div class="unfurl col-md-12" style="display:none;" data-url="<?= $dataurl; ?>"></div>
     <?php if ($vars['object']->canEdit() && (!empty($vars['object']->_id))) { ?>
     <div class="unfurl-edit pull-right small">
-        <a href="#" class="refresh"><i class="fa fa-sync"></i> <?= \Idno\Core\Idno::site()->language()->_('Refresh preview'); ?></a> &nbsp;
-        <a href="#" class="delete"><i class="fa fa-trash"></i> <?= \Idno\Core\Idno::site()->language()->_('Remove preview'); ?></a>
+        <a href="#" class="refresh"><i class="fa fa-sync"></i></a>
```

Comment:
What about having the labels appear as hover titles?

My concern would be that people might confuse rubbish with delete post.

Other than that, LG